packages/frontend/scripts/FirebaseConfigPrinter.js

```diff
@@ -0,0 +1,19 @@
+(async () => {
+  const clipboardy = await import('clipboardy');
+
+  // REPLACE THIS WITH YOUR FIREBASE CONFIG.
+  const firebaseConfig = {
+    apiKey: 'AIzaSyCWgxNPREB1Pw7JXaOjq5c63TdhK0HKHcg',
+    authDomain: 'flat-split.firebaseapp.com',
+    projectId: 'flat-split',
+    storageBucket: 'flat-split.appspot.com',
+    messagingSenderId: '404672973040',
+    appId: '1:404672973040:web:496005cc4f28f56ab75a35',
+    measurementId: 'G-D05DK4ZWJ2',
+  };
+
+  clipboardy.default.writeSync("'" + JSON.stringify(firebaseConfig) + "'");
+  console.log(
+    '🚀 Copied firebase config to clipboard. Paste it inside your .env file as REACT_APP_FIREBASE_CONFIG.',
```

Comment:
Is there any reason that we don't just save firebaseConfig to .env file and push it to remote so that developers do not require to add it on their own?

Reply:
we shouldn't add secrets inside of committed files

Reply:
> Is there any reason that we don't just save firebaseConfig to .env file and push it to remote so that developers do not require to add it on their own?

This is in the env, but other developers (especially the next team) may want their own project and this tool should help them achieve that.

> we shouldn't add secrets inside of committed files
This information isn't secret

Reply:
ah ok cool

Reply:
I see. Thanks for the clarification.